step_definitions/copyCourse-steps.js

```diff
@@ -55,28 +62,28 @@ When(/^.* adds some Material$/, function () {
 	return addEditTopicPage.addMaterial();
 });
 
-When(/^.* adds some Etherpad with name '([^']*)' and description '([^']*)'$/, async function (etherpadName,etherpadDescription) {
+When(/^.* adds some Etherpad with name '([^']*)' and description '([^']*)'$/, async function (
+	etherpadName,
+	etherpadDescription
+) {
 	await addEditTopicPage.addEtherpad(etherpadName, etherpadDescription);
 	await addEditTopicPage.clickCreateTopicButton();
 });
 
-When(/^.* adds some Text '(.*)'$/, async function(text) {
+When(/^.* adds some Text '(.*)'$/, async function (text) {
 	await addEditTopicPage.addText(text);
 });
 
-When(/^.* inputs subtopic name '(.*)'$/, async function(subtopicName) {
+When(/^.* inputs subtopic name '(.*)'$/, async function (subtopicName) {
 	await addEditTopicPage.setSubtopicName(subtopicName);
 	await addEditTopicPage.clickCreateTopicButton();
 });
 
-Then(/^.* course with name '([^']*)' contains topic with name '([^']*)'$/, async function(coursename, topicname) {
+Then(/^.* course with name '([^']*)' contains topic with name '([^']*)'$/, async function (coursename, topicname) {
 	await courseListPage.goToCourses();
 	await courseListPage.isTopicInCourseInSection(coursename, topicname, courseListPage.section.activeCourses);
 });
 
-When(/^.* clicks add-Text-Btn$/, async function() {
-		await addEditTopicPage.clickAddTextBtn()
+When(/^.* clicks add-Text-Btn$/, async function () {
+	await addEditTopicPage.clickAddTextBtn();
 });
-
-	 
-
```

Comment:
This is just automatic code formatting by Prettier. Now it matches our prettier settings.